package.json

```diff
@@ -2,22 +2,24 @@
   "name": "react-dag",
   "version": "2.0.0",
   "description": "This is a base implementation of wrapping jsplumb with react + redux to be more usable in a react based environment",
-  "main": "dist/react-dag.js",
+  "main": "./dist/react-dag.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "dev-build": "webpack --watch -d",
+    "dev-build-watch": "webpack --watch -d",
+    "dev-build": "webpack -d",
     "prod-build": "NODE_ENV=production webpack -p",
     "lib-test-build": "NODE_ENV=libtest webpack -d"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "babel-runtime": "^6.11.6",
     "classname": "0.0.0",
-    "jsplumb": "sporritt/jsPlumb#2.2.0",
+    "jsplumb": "git://github.com/sporritt/jsPlumb#2.2.0",
```

Comment:
hmm I needed it to build on my computer, could be my issue not sure.  It was not pulling the jsplumb dependencies at all.